examples/todomvc/src/actions/index.js

```diff
@@ -4,5 +4,6 @@ export const addTodo = text => ({ type: types.ADD_TODO, text })
 export const deleteTodo = id => ({ type: types.DELETE_TODO, id })
 export const editTodo = (id, text) => ({ type: types.EDIT_TODO, id, text })
 export const completeTodo = id => ({ type: types.COMPLETE_TODO, id })
-export const completeAll = () => ({ type: types.COMPLETE_ALL })
+export const completeAllTodos = () => ({ type: types.COMPLETE_ALL_TODOS })
```

Comment:
Why the rename? Seems like the context is obvious here regardless.

Reply:
Yes, it is obvious. I just feel having `Todos` attached to the `completeAll` will make it much more explicit and consistent with the others.